components/x-gift-article/src/AdvancedSharingOptions.jsx

```diff
@@ -23,17 +14,12 @@ export const AdvancedSharingOptions = (props) => {
 		}
 	}
 
-	const includeHighlightsHandler = (event) => {
-		actions.setIncludeHighlights(event.target.checked)
-	}
-
 	return (
 		<div>
 			<div
 				className="o-forms-field o-forms-field--optional o-forms-field--professional share-article-dialog__advanced-sharing-options"
 				role="group"
-				aria-labelledby="radio-group-title"
-			>
+				aria-labelledby="radio-group-title">
```

Comment:
**Change request:** Sorry I think you might be getting unlucky here but your change highlighting an already existing issue.

The `aria-labelledby` attribute seems to be pointing to an non-existence element. I can't see anything in this repo with an `id` of `radio-group-title`

Reply:
I think we can probably just remove the attribute

Reply:
we probably removed the element with that Id when we updated the gift article component a some time ago , will check the commit history for more info and remove this lable for now